website/docs/reference/resource-configs/firebolt-configs.md

```diff
@@ -0,0 +1,299 @@
+---
+title: "Firebolt configurations"
+id: "firebolt-configs"
+---
+
+## Using dbt-firebolt
+
+### `quote_columns`
+
+To prevent a warning, you should add a configuration as below to your `dbt_project.yml`. For more info, see the [relevant dbt docs page](https://docs.getdbt.com/reference/resource-configs/quote_columns).
+
+
+<File name>
+```yml
+seeds:
+  +quote_columns: false  #or `true` if you have csv column headers with spaces
+```
+</File>
```

Comment:
do we still need this section on `quote_columns`? my vote is to strike it.

something looks weird w/ it right now anyway

cc: @kevinmarr @amychen1776 @ericf-firebolt 

<img width="685" alt="image" src="https://user-images.githubusercontent.com/8158673/145123522-c9af34e8-25e0-445c-9d3d-882146630f7c.png">


```suggestion
```

Reply:
Since this in the profiles config page, I don't think so.

Reply:
Btw, the thing that was weird is that it said "yml" after the triple ticks and not "yaml".